gbpservice/nfp/configurator/agents/loadbalancer_v2.py

```diff
@@ -0,0 +1,979 @@
+    def get_logical_device(self, pool_id, context):
+        """ Calls data filter library to get logical device from pool_id.
+
+        :param pool_id: object type
+        :param context: context which has list of all pool related resources
+                        belonging to that tenant
+
+        Returns: logical_device
+        """
+        return self.call(
+            context,
+            self.make_msg(
+                'get_logical_device',
+                pool_id=pool_id
+            )
+        )
+
+    def update_status(self, obj_type, obj_id, root_lb_id,
+                      provisioning_status, operating_status,
+                      agent_info, obj=None):
+        """ Enqueues the response from LBaaS V2 operation to neutron plugin.
+
+        :param obj_type: object type
+        :param obj_id: object id
+        :param root_lb_id: root loadbalancer id
+        :param provisioning_status: an enum of (‘ACTIVE’, ‘PENDING_CREATE’,
+        ‘PENDING_UPDATE’, ‘PENDING_DELETE’, ‘ERROR’)
+        :param operating_status:  an enum of
+        (‘ONLINE’, ‘OFFLINE’, ‘DEGRADED’, ‘ERROR’)
+
+        """
+
+        msg = {'info': {'service_type': lb_constants.SERVICE_TYPE,
+                        'context': agent_info['context']},
+               'notification': [{'resource': agent_info['resource'],
+                                 'data':{'obj_type': obj_type,
+                                         'obj_id': obj_id,
+                                         'notification_type': 'update_status',
+                                         'root_lb_id': root_lb_id,
+                                         'provisioning_status':
+                                             provisioning_status,
+                                         'operating_status':
+                                             operating_status,
+                                         obj_type: obj}}]
+               }
+        self.notify._notification(msg)
+
+    # TODO(jiahao): copy from v1 agent need to review
+    def update_pool_stats(self, pool_id, stats, context, pool=None):
+        """ Enqueues the response from LBaaS operation to neutron plugin.
+
+        :param pool_id: pool id
+        :param stats: statistics of that pool
+
+        """
+        msg = {'info': {'service_type': lb_constants.SERVICE_TYPE,
+                        'context': context.to_dict()},
+               'notification': [{'resource': 'pool',
+                                 'data': {'pool_id': pool_id,
+                                          'stats': stats,
+                                          'notification_type': (
+                                                        'update_pool_stats'),
+                                          'pool': pool_id}}]
+               }
+        self.notify._notification(msg)
+
+    # TODO(jiahao): copy from v1 agent need to review
+    def vip_deleted(self, vip, status, agent_info):
```

Comment:
I don't think this is needed as LBaaS V2 resource don't have VIP. Can you check in LBaaS V2 neutron plugin and plugin drivers if this callback is expected? If it is not there, this can be removed.